### ISSUE TITLE
ExceptionHandler: Cannot assign to read only property

### ISSUE THREAD

**Issue by dennyreiter** [mKeRix/room-assistant#366]
**Describe the bug**
Room Assistant no longer starts, it encounters an exception after loading the intergrations and while trying to start the Nest application

**To reproduce**
Install room-assistant and start.
**Relevant logs**
*** npm install ***
```
npm WARN @nestjs/mapped-types@0.1.1 requires a peer of class-transformer@^0.3.0 but none is installed. You must install peer dependencies yourself.
npm WARN @nestjs/mapped-types@0.1.1 requires a peer of class-validator@^0.11.1 || ^0.12.0 but none is installed. You must install peer dependencies yourself.
```

*** room-assistant -v***
```
*** WARNING *** The program 'node' uses the Apple Bonjour compatibility layer of Avahi.
*** WARNING *** Please fix your application to use the native API of Avahi!
*** WARNING *** For more information see <http://0pointer.de/blog/projects/avahi-compat.html>
*** WARNING *** The program 'node' called 'DNSServiceRegister()' which is not supported (or only supported partially) in the Apple Bonjour compatibility layer of Avahi.
*** WARNING *** Please fix your application to use the native API of Avahi!
*** WARNING *** For more information see <http://0pointer.de/blog/projects/avahi-compat.html>
11/25/2020, 12:08:53 PM - info - IntegrationsModule: Loading integrations: home-assistant, grid-eye
11/25/2020, 12:08:57 PM - info - NestFactory: Starting Nest application...
11/25/2020, 12:08:57 PM - error - ExceptionHandler: Cannot assign to read only property 'hasOwnProperty' of object '#<Object>'
```

**Relevant configuration**
Paste the relevant parts of your configuration below.

```
global:
  instanceName: mediaroom
  integrations:
    - homeAssistant
    - gridEye
homeAssistant:
  mqttUrl: 'mqtt://192.168.0.77:1883'
  mqttOptions:
    username: youruser
    password: yourpass
gridEye:
  deltaThreshold: 3
cluster:
  networkInterface: wlan0
  port: 6425
  quorum: 3

```

**Expected behavior**
Room Assistant was running, just with Grid-Eye but the response seemed slow. I did an upgrade and it started spewing unicode garbage characters out to MQTT.  After trying to downgrade to 2.10.1, Grid-Eye didn't seem to do anything.  I have uninstalled and reinstalled, even deleted /opt/nodejs to try and start fresh. The install does complain about missing peer dependencies for nestjs, which I install, but they don't seem to 'stick'

**Environment**
 - room-assistant version: 2.12.0
 - installation type: NodeJS
 - hardware: Raspberry Pi Zero W
 - OS: Linux

**Additional context**

**Comment by mKeRix** [comment]
The bug that you are describing was fixed in v2.11.2, it occurs since a library that room-assistant depends on changed how they use the settings object in a minor upgrade. You can either try to downgrade the `mqtt` dependency or upgrade room-assistant to at least v2.11.2 again (latter is recommended).

The peer dependency stuff you can ignore, the warnings are not really relevant for any room-assistant usage but I also can't disable them since they come from a dependency. 

The unicode characters are just the log of the camera state, it tries to convert the image data into a string. I know it's not pretty and makes the verbose log unreadable, I just haven't gotten around to disabling state logging for camera entities yet.

And another quick sidenote, since you are using Grid-EYE on a Pi Zero: starting with v2.12.0 the heatmap generation was changed and will be a bit slower, but work without native dependencies or memory leaks. v2.13.0-beta.2 adds a config optio that allows you to disable the heatmap should you not need it and would rather have better performance.

**Comment by dennyreiter** [comment]
Thank you.  Yes, the unicode characters it finally dawned on me what they were, and just excluding them from my mqtt_subscribe took care of that.  

Thank you for all your work on this, you are a rock star!
